force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -322,17 +398,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer getSalary(String employeeName) {
         // Initialize a Map
-
+        Map<String, Integer> Salariesofemployee = new Map<String, Integer>();
         // Add key-value pairs to the Map
-        // add John Doe => 50000
-        // add Jane Smith => 60000
-        // add Sam Brown => 55000
-        // add Alice Johnson => 65000
+        Salariesofemployee.put('John Doe', 50000);
```

Comment:
(s) for Maps, follow the convention of `valueByName` for your variables and use camelCase for all variables --> `Map<String, Integer> salaryByEmployee...`